FE-service/src/components/subpages/arena/WinnerModal.jsx

```diff
@@ -0,0 +1,46 @@
+import { createPortal } from "react-dom";
+import { Typography } from "@mui/material";
+import {
+  ModalOverlay,
+  ModalContainer,
+  ModalContent,
+  LeaveButton,
+  PokemonName,
+  PokemonImage,
+} from "./WinnerModal.styles";
+import { useTheme } from "../../../context/ThemeContext";
+import { POKEMON_IMG } from "../../../../../apiConfig";
+
+const WinnerModal = ({ winner, onClose }) => {
+  const handleClose = () => {
+    onClose();
+  };
+  const { theme } = useTheme();
```

Comment:
We place hook calls below the application's state at the very top of the component.